fastly/terraform/modules/service-vcl/vcl/fastly-devhub-globe.vcl

```diff
@@ -55,7 +58,8 @@ sub vcl_log {
       substr(fastly_info.state, 0, 1) " "
       regsuball(req.proto, "[^\d.]", "") " "
       regsuball(tls.client.protocol, "[^\d.]", "") " "
-      table.lookup(globeviz_known_browsers, client.browser.name, "Z")
+      table.lookup(globeviz_known_browsers, client.browser.name, "Z") " "
+      if (client.socket.conn_type == "mobile", "M", "-")
```

Comment:
it's located at `client.geo.conn_type` -- https://developer.fastly.com/reference/vcl/variables/geolocation/client-geo-conn-type/